appveyor.yml

```diff
@@ -7,8 +7,8 @@ assembly_info:
   patch: true
   file: '**\AssemblyInfo.cs'
   assembly_version: '4.0.0'
-  assembly_file_version: '4.7.1.{build}' #NLog.Web
-  assembly_informational_version: '4.7.1' #NLog.Web
+  assembly_file_version: '4.8.0.{build}' #NLog.Web
```

Comment:
4.8 because the update of the dependency?  So skip 4.7.2?

Reply:
Should I keep NLog.Web at 4.7.1 ?

Reply:
Reverted version change for NLog.Web